### PR TITLE
Added missing strings in english.xml

### PR DESCRIPTION
**Pull request by rddim**
-Added missing strings in english.xml for better translation
-Replace `r\` with `&#x0A;` for proper new line(s)

### CODE REVIEW

File: PowerEditor/installer/nativeLang/english.xml [hunk 1]
```diff
@@ -904,21 +904,30 @@
...
             <FileAlreadyOpenedInNpp title="" message="The file is already opened in the Notepad++."/>
             <DeleteFileFailed title="Delete File" message="Delete File failed"/>
 
             <!-- $INT_REPLACE$ is a place holder, don't translate it -->
-            <NbFileToOpenImportantWarning title="Amount of files to open is too large" message="$INT_REPLACE$ files are about to be opened.\rAre you sure to open them?"/>
-			<SettingsOnCloudError title="Settings on Cloud" message="It seems the path of settings on cloud is set on a read only drive,\ror on a folder needed privilege right for writing access.\rYour settings on cloud will be canceled. Please reset a coherent value via Preference dialog."/>
+            <NbFileToOpenImportantWarning title="Amount of files to open is too large" message="$INT_REPLACE$ files are about to be opened.&#x0A;Are you sure to open them?"/>
+			<SettingsOnCloudError title="Settings on Cloud" message="It seems the path of settings on cloud is set on a read only drive,&#x0A;or on a folder needed privilege right for writting access.&#x0A;Your settings on cloud will be canceled. Please reset a coherent value via Preference dialog."/>
```
Comment (SinghRajenM) [inline PowerEditor/installer/nativeLang/english.xml]: writting? writing is proper.

Comment (rddim) [inline PowerEditor/installer/nativeLang/english.xml]: Oh, yes, you are right, it's my fault. Thanks

**Comment by SinghRajenM** [comment]
> -Replace  r\ with &#x0A; for proper new line(s)

If so , then \r\n should be used as it is standard for windows. Same is followed in scintilla.
Refer link  - 

- [wiki](https://en.wikipedia.org/wiki/Newline)
- [msdn](https://social.msdn.microsoft.com/Forums/en-US/47af2197-26b4-4b9e-90e8-bfa9d5cd05b4/what-is-the-deference-between-r-n-and-rn-?forum=csharplanguage)

**Comment by rddim** [comment]
You are probably right. I make some test and at this point the results are:

`r\`

npp-r

`r\n\`

npp-rn

`&#x0A;`

npp-x0a

File: PowerEditor/installer/nativeLang/english.xml (hunk 1 above)
Comment (donho) [inline PowerEditor/installer/nativeLang/english.xml]: @SinghRajenM Your our spellchecker. Damn, *You're ! :)

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/193ecfd6dfaa2c37e0d9aa5604bed7568a6c316f

**Pull request closed without merging**